modules/gltf/extensions/physics/gltf_physics_body.h

```diff
@@ -47,6 +47,7 @@ class GLTFPhysicsBody : public Resource {
 	real_t mass = 1.0;
 	Vector3 linear_velocity = Vector3();
 	Vector3 angular_velocity = Vector3();
+	Vector3 center_of_mass = Vector3();
```

Comment:
Vector3 is default initialized, so those three assignments are not needed.
Doesn't hurt either if that's to make it very explicit in a context where some values have non-default initializers.

Reply:
Hmm, either way works but I think it's good to explicitly indicate that there is a default value of 0, and that the code is not expected to always put values in it.